### PR TITLE
feat(standards): add graphics roles

### PR DESCRIPTION
**Pull request by TwoLaid**
Add support for roles of the WAI-ARIA Graphics Module: https://www.w3.org/TR/graphics-aria-1.0/#introduction

* Adds three new roles: `graphics-document`, `graphics-object`, `graphics-symbol`
* Adjusts existing standards test
* Extends Typescript types for the new roles
* Adds documentations for the new roles

Closes issue: https://github.com/dequelabs/axe-core/issues/2684 and https://github.com/dequelabs/axe-core/issues/1189.

### CODE REVIEW

File: axe.d.ts [hunk 1]
```diff
@@ -162,7 +167,7 @@ declare namespace axe {
     unsupported?: boolean;
   }
   interface AriaRoles {
-    type: AriaRolesType | DpubRolesType;
```
Comment (straker) [inline axe.d.ts:165]: I don't think this is correct. The `type` should be the `type` property from the role, which is just `structure` for each one. So you shouldn't need a new type for this as `AriaRoleTypes` covers strcuture.

**Review by straker — CHANGES_REQUESTED** [review]
Nice work! I'll let @WilcoFiers do a final review on this one to make sure we're not missing something.

File: axe.d.ts [hunk 2]
```diff
@@ -162,7 +167,7 @@ declare namespace axe {
...
+    type: AriaRolesType | DpubRolesType | GraphicsRolesType;
```
Comment (straker) [inline axe.d.ts:170]: I don't think this is correct. The type should be the type property from the role, which is just structure for each one. So you shouldn't need a new type for this as AriaRoleTypes covers strcuture.

Comment (TwoLaid) [reply axe.d.ts]: Yes, you're absolutely right. I misunderstood that bit. Removing it now.

**Review by WilcoFiers — APPROVED** [review]

**Comment by WilcoFiers** [comment]
@TwoLaid Thank you so much for the contribution! Much appreciated.

**Comment by WilcoFiers** [comment]
Security app isn't responding. -- I reviewed this PR for security.

**Pull request merged**